compiler/compiler.go

```diff
@@ -2182,21 +2182,31 @@ func (c *Compiler) parseExpr(frame *Frame, expr ssa.Value) (llvm.Value, error) {
 
 		// Bounds checking.
 		if !frame.fn.IsNoBounds() {
-			if sliceLen.Type().IntTypeWidth() < c.uintptrType.IntTypeWidth() {
+			checkFunc := "sliceBoundsCheckMake"
+			biggestInt := c.uintptrType
+			biggestIntWidth := biggestInt.IntTypeWidth()
```

Comment:
the proper name would be "pointerWidth", uintptr is not a "biggest int"

Reply:
No, because it can be wider than a pointer.

Reply:
Umm, `c.uintptrType` is supposed to be as wide as a pointer; but it is definitely not the "biggest int".

Reply:
Yes, `c.uintptrType` is the same thing as `uintptr` and thus as big as a pointer. But a few lines below it is set to `uint64`, which certainly can be bigger than a pointer:
https://github.com/aykevl/tinygo/pull/129/files/4992faa0e881d0ea40d6cf8abf81d3f96e6344e3#diff-aa244bd5dbd5c4c8e88f1ad97ce253ebR2192
So `pointerWidth` is certainly wrong. Let me know if you have an idea for a better variable name.

Reply:
Those names `capacityType` and `capacityTypeWidth` seem like a pretty good change to me.

Reply:
Not entirely happy with the other name but it's okay, change made to avoid more bikeshedding.